.github/workflows/staticsite.yml

```diff
@@ -45,7 +45,7 @@ jobs:
         # If GH_GIST_TOKEN doesn't exist, empty value gets passed.
         # Unauthenticated call gets made to Gist API which can result in rate limits,
         # Especially in Github Actions
-        run: /bin/bash -c 'npm install;npm run build --ghtoken=${{ secrets.GH_GIST_TOKEN }}'
+        run: /bin/bash -c 'npm install;npm run build --gisttoken=${{ secrets.GH_GIST_TOKEN }} --githubtoken=${{ secrets.GITHUB_TOKEN }}'
```

Comment:
Apparently the GITHUB_TOKEN generated in Github Actions only has repo permissions, no gist permissions. So a separate token is required.